multicluster_setup.adoc

```diff
@@ -0,0 +1,139 @@
+Switched to context "host-admin".
+https://api.<host_cluster_name>.devcluster.openshift.com:6443
+
+$ oc config use-context member-admin && oc whoami --show-server
+Switched to context "member-admin".
+https://api.<member_cluster_name>.devcluster.openshift.com:6443
+
+----
+
+Once both contexts are configured, we can move on with the operator deployments, starting with the Host operator.
+
+[source,bash]
+----
+
+$ export HOST_NS=toolchain-host-operator
+$ export REGISTRATION_SERVICE_NS=${HOST_NS}
+
+# deploy in the specific `toolchain-host-operator` namespace
+$ oc config use-context host-admin && HOST_NS=${HOST_NS} make get-host-and-reg-service deploy-host
+----
+
+This will clone the remote https://github.com/codeready-toolchain/host-operator.git and https://github.com/codeready-toolchain/registration-service.git repositories in `/tmp/codeready-toolchain/` and build the operator and service from these temporary directories, then deploy the Host operator on the Host cluster.
+
+After a minute (or more), both `host-operator` and `registration-service` pods should be available:
+
+[source,bash]
+----
+$ oc get pods -n ${HOST_NS}
+NAME                                    READY   STATUS    RESTARTS   AGE
+host-operator-...-...                   1/1     Running   0          ...
+registration-service-...-...            1/1     Running   0          ...
+registration-service-...-...            1/1     Running   0          ...
+----
+
+== Member Cluster Setup
+
+The same logic as above is applied to deploy the Member operator on the Member cluster:
+
+[source,bash]
+----
+$ export MEMBER_NS=toolchain-member-operator
+
+# on the member cluster, in the `toolchain-member-operator` namespace
+$ oc config use-context member-admin && MEMBER_NS=${MEMBER_NS} make get-member-operator-repo deploy-member
+...
+
+$ oc get pods -n ${MEMBER_NS}
+NAME                               READY   STATUS    RESTARTS   AGE
+member-operator-...-...            1/1     Running   0          ...
+----
+
+== Configuring the Operators
+
+Once Host and Member operators have been deployed on their respective cluster, they need to be configured to communicate with each other.
+
+At this stage, each operator has its own status resource, which is in a `Ready=False` status, because the host-member connection is not set yet:
+
+[source,bash]
+----
+# on the host cluster
+$ oc config use-context host-admin && oc get toolchainstatus -n ${HOST_NS}
+NAME               READY   LAST UPDATED
+toolchain-status   False   ...
+
+# on the member cluster
+$ oc config use-context member-admin && oc get memberstatus -n ${MEMBER_NS}
+NAME                      READY   LAST UPDATED
+toolchain-member-status   False   ...
+----
+
+[source,bash]
+----
+## on the member cluster
+
+# retrieve the API endpoint of the member cluster
+$ oc config use-context member-admin && oc get infrastructure cluster -o jsonpath='{.status.apiServerURL}' 
+https://api.<member_cluster_name>.devcluster.openshift.com:6443  
+
```

Comment:
I guess that we don't need this part either
```suggestion
```

Reply:
ah right 🤦‍♂️